<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>peoplegame</class>
 <widget class="QMainWindow" name="peoplegame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushStart">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;，rgb(221, 29, 255)；</string>
    </property>
    <property name="text">
     <string>得分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;，rgb(221, 29, 255)；</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStart_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>重来</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 20pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>请输入你的名字</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;黑体&quot;rgb(222, 137, 255);</string>
    </property>
    <property name="text">
     <string>人   品   计   算   器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_word">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;，rgb(221, 29, 255)；</string>
    </property>
    <property name="text">
     <string>建议：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;，rgb(221, 29, 255)；</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;，rgb(221, 29, 255)；</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;，rgb(221, 29, 255)；</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="userBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>✔</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushStart</zorder>
   <zorder>label_scor</zorder>
   <zorder>label_name_4</zorder>
   <zorder>pushStart_2</zorder>
   <zorder>lineName</zorder>
   <zorder>label</zorder>
   <zorder>label_word</zorder>
   <zorder>label_name_3</zorder>
   <zorder>label_name</zorder>
   <zorder>label_name_2</zorder>
   <zorder>userBtn</zorder>
   <zorder>commandLinkButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
